### PR TITLE
Feat: added Image and Input Components

### PR DESCRIPTION
**Pull request by ArjunGTX**
Added image and Input
ezgif-2-f50a762fa9

### CODE REVIEW

File: css/components/input.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
+@import url("https://fonts.googleapis.com/css2?family=Nunito:wght@400;500;600;700;800&display=swap");
+@import url("../utility.css");
+
+.input {
+    background-color: inherit;
+    border: none;
+    padding: 0.625rem;
+    padding-bottom: 3px;
+    margin: 0.625rem;
```
Comment (nerdyspook) [inline css/components/input.css]: you can set a global variable for spacing as it seems to repeat a few times

**Review by nerdyspook — COMMENTED** [review]
It looks good

File: css/components/input.css (hunk 1 above)
Comment (ArjunGTX) [reply css/components/input.css]: Sure I will.

**Review by tanishk15sharma — APPROVED** [review]
apart from above suggestion , looks good to me

File: css/components/input.css (hunk 1 above)
Comment (ArjunGTX) [reply css/components/input.css]: I've replaced the repeated values with variables

**Review by SukanyaSen530 — APPROVED** [review]
GTG 😊

File: css/components/input.css (hunk 1 above)
Comment (nerdyspook) [reply css/components/input.css]: Yeah you can resolve this issue now.

**Pull request merged**